lightning/src/routing/router.rs

```diff
@@ -481,14 +481,16 @@ pub struct RouteParameters {
 	/// This limit also applies to the total fees that may arise while retrying failed payment
 	/// paths.
 	///
-	/// Default value: `None`
+	/// Note that values below a few sats may result in some paths being spuriously ignored.
 	pub max_total_routing_fee_msat: Option<u64>,
 }
 
 impl RouteParameters {
 	/// Constructs [`RouteParameters`] from the given [`PaymentParameters`] and a payment amount.
+	///
+	/// [`Self::max_total_routing_fee_msat`] defaults to 1% of the payment amount + 50 sats
 	pub fn from_payment_params_and_value(payment_params: PaymentParameters, final_value_msat: u64) -> Self {
-		Self { payment_params, final_value_msat, max_total_routing_fee_msat: None }
+		Self { payment_params, final_value_msat, max_total_routing_fee_msat: Some(final_value_msat / 100 + 50_000) }
```

Comment:
Shouldn't we set this in `OutboundPayments::create_pending_payment` instead of here? That currently sets a default if not set.

Reply:
Oh, right. I misread that. Would it make more sense to set it there, though, rather than modifying the `RouteParameters` upon construction? I guess not as the user wouldn't be able to pass `None` then.